NCIOCPL/ncids
# (#437) Fix header buttons not importing styles correctly

Closes #437 

Findings
From what I can gather, extending `.usa-button` has a lot of unintended consequences when it comes to responsive styling. 

How to test
Locally, boot up Storybook and head to the Inner content page (http://127.0.0.1:6006/?path=/story/templates-inner-content--inner-content)
Moving the order of the header import should not break header. Previously, this file depended on header being imported last.
